docker/Dockerfile_ubuntu_18.04

```diff
@@ -0,0 +1,27 @@
+FROM nvidia/cudagl:9.2-runtime-ubuntu18.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    python3 python3-dev ipython3 module-init-tools curl build-essential python3-pip
+
+RUN pip3 install -U pip setuptools wheel
+
+RUN pip3 install numpy posix_ipc holodeck pytest opencv-python
+
+RUN adduser --disabled-password --gecos "" holodeckuser
+
+RUN echo holodeckuser:holodeck | chpasswd
+
+# This should be COPY ../ but docker doesn't allow copying files outside the context
+# for this to copy the project files either run the build command in this directory with the
+# previous directory as the context: docker build -t pccl/holodeck[:tag] -f ./Dockerfile ..
+# or run it from the parent directory and provide the docekr file location
+# docker build -t pccl/holodeck[:tag] -f ./docker/Dockerfile .
+COPY ./ /home/holodeckuser/source/holodeck/
```

Comment:
You can use the [WORKDIR](https://docs.docker.com/engine/reference/builder/#workdir) directive

Reply:
Done